src/imitation/scripts/analyze.py

```diff
@@ -123,7 +126,13 @@ def gather_tb_directories() -> dict:
                 os.makedirs(symlinks_dir, exist_ok=True)
 
                 tb_symlink = osp.join(symlinks_dir, run_name)
-                os.symlink(tb_src_dir, tb_symlink)
+                try:
```

Comment:
Since this symlink-plus-warning is used in two places in the code, could consider factoring it out into a utility function, though it's simple enough not sure it's worth it. (Would be if we use it in 3 or more places.)